wpeverest/user-registration
# Fix - class translation issue

### All Submissions:


### Changes proposed in this Pull Request:
Previously, when the users Translate the fields option label in any other languages it shows error in some field .This PR solve this issue

### How to test the changes in this Pull Request:

1. Install loco translate plugin and now navigate to the form builder.
2. Now, drag and drop the fields like (select, radio, checkbox) and add more than 3 choices in case of these fields.
3. Now translate the field option (like field name, options, label) of the respective field and update the form.
4. Navigate to the frontend of the form and verify whether the options are showing more than 3 or not.

PR For Cases to test
https://github.com/wpeverest/user-registration-advanced-fields/pull/21
https://github.com/wpeverest/user-registration-file-upload/pull/12

### Types of changes:


### Other information:


### Changelog entry

> Fix - class translation issue
